content/en/blog/news/cjxd-202003.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "CloudBees contributions to Jenkins X - March 2020"
+date: 2020-03-23T00:00:00-00:00
+draft: false
+description: >
+  CloudBees contributions to Jenkins X - March 2020
+categories: [blog]
+keywords: [Jenkins X,Community,2020]
+slug: "cjxd-march-20"
+aliases: []
+author: David Conde
+---
+
+<figure>
+<img src="/images/logo/cloudbees.png"/>
+</figure>
+
+March has been a really busy month for us working on Jenkins X, several exciting projects are now well underway. 
+
+Recently at office hours we discussed a [proposal to improve how we provision](https://github.com/jenkins-x/enhancements/tree/master/proposals/3) the cloud resources (storage buckets, service accounts & cryptographic keys) that Jenkins X requires.
+The idea behind this enhancement is that some companies may require the infrastructure to be provisioned by a user (someone in operations perhaps) that requires a different set of permissions than the user that runs `jx boot` to install Jenkins X.
+Previously this happened when a user ran `jx boot`. But from community feedback we learned that some users would like this process split. 
+To help with this we are in the process of developing terraform modules that you can use to create the necessary resources. 
+Work on this has been moving really quickly and we have been focusing on support for GKE, EKS & AKS.
```

Comment:
would be good to link to the repos and share how the OSS community can get involved

Reply:
any links to docs / blogs / demos etc might help too